patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -273,7 +273,23 @@
          if (p_71059_1_.func_70075_an())
          {
              if (!p_71059_1_.func_85031_j(this))
-@@ -1203,9 +1283,11 @@
+@@ -1172,14 +1252,7 @@
+                 float f = (float)this.func_110148_a(SharedMonsterAttributes.field_111264_e).func_111126_e();
+                 float f1;
+ 
+-                if (p_71059_1_ instanceof EntityLivingBase)
+-                {
+-                    f1 = EnchantmentHelper.func_152377_a(this.func_184614_ca(), ((EntityLivingBase)p_71059_1_).func_70668_bt());
+-                }
+-                else
+-                {
+-                    f1 = EnchantmentHelper.func_152377_a(this.func_184614_ca(), EnumCreatureAttribute.UNDEFINED);
+-                }
```

Comment:
sounds strange to do some unnecessary calculations, but it decreases patch size, so ok.